spicy/lib/protocols/tftp.spicy

```diff
@@ -0,0 +1,91 @@
+#
+# Trivial File Transfer Protocol
+#
+# Specs from https://tools.ietf.org/html/rfc1350
+#
+
+module TFTP;
+
+import spicy;
+
+# Common header for all messages:
+#
+#      2 bytes
+# ---------------
+# |  TFTP Opcode  |
+#  ---------------
+
+public type Message = unit {
```

Comment:
Let's add a comment here why this type is `public` even though we use other types from this module as well.